contracts/vyper/Erc20.vy

```diff
@@ -24,7 +24,7 @@ totalSupply: public(uint256)
 minter: address
 
 
-@external
+@deploy
```

Comment:
Did you check why this changes need? Maybe vyper version is newer?

Reply:
Yes, it's due to the newer versions, we're testing now with beta vyper, such as 4beta4 (Kristina will create new issue to handle this gracefully).
https://pypi.org/project/vyper/#history